var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -129,26 +129,28 @@ def install_targets(self):
         ]
         return make_args + self.make_defs
 
-    @run_after('install')
-    def check_install(self):
-        spec = self.spec
```

Comment:
either way, do NOT comment out this method, that's hardly a solution to the problem.